Kajabi/sage-lib
# [Sage-755] Include Figma embed within Docs

Closes #755 

## Description

Create a link to the Figma embed for all components available in the designs within Figma.


## Screenshots

|  Before  |  After  |
|--------|--------|
| ![Screen Shot 2021-09-01 at 1 32 44 PM](https://user-images.githubusercontent.com/24237393/131724962-d387a9fd-c0ef-404a-a9a3-2c941dc3951c.png) | ![tab-open](https://user-images.githubusercontent.com/24237393/131724884-0db6ae82-0185-4263-865e-ead0aa7adcee.png) |


## Testing in `sage-lib`

1. Navigate to the [Alerts](http://localhost:4000/pages/component/alert)
2. Click the "Figma Designs" tab
3. Observe the embed within that tab content

## Testing in `kajabi-products`
1. (**LOW**) No impact on `kajabi-products`
